src/main.ts

```diff
@@ -117,6 +117,24 @@ if (require('electron-squirrel-startup')) {
 
 initializeLogger()
 
+// Used to convert `.json` files into the appropriate file extension for the Generator
+async function migrateJsonGenerator() {
+  const files = (
+    await readdir(GENERATORS_PATH, { withFileTypes: true })
+  ).filter((f) => f.isFile() && path.extname(f.name) === '.json')
```

Comment:
Separating the call to `readdir` from the filtering really helps with the readability.

```suggestion
  const items = await readdir(GENERATORS_PATH, { withFileTypes: true })
  const files = items.filter((f) => f.isFile() && path.extname(f.name) === '.json')
```

Then again, I suppose we don't expect this to live long after GA.

Reply:
Nothing against making things more readable :)